nixie/unittests/test_render_diagram.py

```
@@ -21,12 +21,13 @@ async def test_render_diagram_writes_file_and_logs(
     path = Path("doc.md")
     block = "A-->B"
 
-    async def fake_create_subprocess_exec(*cmd: str, **kwargs: object) -> object:
+    async def fake_create_subprocess_exec(*cmd: str, **_kwargs: object) -> object:
```

Ruff (0.12.2)

Unused function argument: `cmd`

(ARG001)